.github/dependabot.yml

```diff
@@ -4,3 +4,7 @@ updates:
     directory: "/"
     schedule:
       interval: daily
+  - package-ecosystem: "docker"
```

Comment:
nitpicking, if it works with removed quotes then use just docker to be aligned with how maven is written above.